### DISCUSSION TITLE
Components which will inform 1 of 5 but when using Tab button

### DISCUSSION THREAD

**Discussion by Ruslan-I7** [adobe/react-spectrum#6869] [Q&A]
Hi there,

The question is, are there some components that can inform text like that "1 of 5"  when using the Tab button? Or is it non-standard behavior?

**Comment by snowystinger** [comment]
You could add your own aria-label? Otherwise, this sounds pretty specific to whatever you're building.

The closest thing might be a step list? We have one in alpha in the react spectrum packages, but we haven't done something like it in RAC or the hooks yet.

What is it that you're building?

**Comment by Ruslan-I7** [comment]
Yes, I can add my own prop, I was just wondering, maybe there is something what was already done in the lib. Like when we use [ListBox](https://react-spectrum.adobe.com/react-aria/ListBox.html) it says us a current position

It's a simple carousel without autoplay and stuff, moves have to be done through the Tab button

**Reply by snowystinger** [reply]
ListBox is following a specific aria pattern for saying the current position and it follows arrow key navigation. We haven't built a carousel yet. I think this is something you'll need to implement yourself with aria-label.
